notebooks/5_local_search/index.md

```diff
@@ -0,0 +1,469 @@
+# Local Search
```

Comment:
PS: 
- Consider a clickable table of contents
- Consider moving the pictures to center